membership.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Membership
+layout: post
+---
+
+## SKIP Student Association Structure Overview
+
+The SKIP Student Association operates in collaboration with Göta Studentkår and is designed to enhance the student experience by organizing events. To effectively achieve its mission, SKIP comprises two distinct categories of members: "Board Members" and "Regular Members." It's important to note that all members, both board and regular, must also be members of Göta Studentkår.
+
+![Skip structure](/assets/imgs/skip-structure.png)
```

Comment:
Good! I'm just wondering that it doesn't look too well under the white background, perhaps I'll wrap the diagram in a rectangle with some border shadow, or what not!

Reply:
Or, alternatively, we could change the theme; the darker themes look more appealing anyway!

Reply:
Thats up to you! But I do agree that the dark theme looks better :)

Reply:
I tried out the different themes, and night definitely suited best imo.

Reply:
On it!

Reply:
I added the file in this PR, so if it gets deployed it should be all good

Reply:
I do agree! Looks great.